lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -33,13 +33,23 @@ class HighChart
     #
     def to_html(placeholder=random_canvas_id)
       chart_hash_must_be_present
-      high_chart(placeholder, self)
+      # Helps to denote either of the three classes.
+      chart_class = extract_chart_class
+      # When user wants to plot a HighMap
+      if chart_class == 'Map'
+        high_map(placeholder, self)
+      # When user wants to plot a HighStock
+      elsif chart_class == 'StockChart'
+        high_stock(placeholder, self)
+      # No need to pass any value for HighChart
+      else
```

Comment:
I think it must be specific 'Chart'. If user have entered some `xyz` chart_class then raise error 'Not implemented' or '3 chart_class chart, stock, map must be selected ' something like that.